docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -107,7 +107,7 @@ to the relevant host(s). Consider the following inventory group:
 
 You can create `group_vars/gatewayed.yml` with the following contents::
 
-    ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -q user@gateway.example.com"'
+    ansible_ssh_common_args='-o ProxyCommand="ssh -W %h:%p -q user@gateway.example.com"'
```

Comment:
Using `=` instead of `:` in YAML would result in syntax error.

Can you be a more specific as to why you think this change is needed?

Reply:
Sorry, I was not that specific, but I think a note should be great, at YAML you should use colon, but if you use hosts file (for example using /etc/ansible/hosts) you have to use equal sign.